gapis/gfxapi/synchronization/synchronization_data.go

```diff
@@ -12,13 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package gfxapi
+package synchronization
 
-import "sort"
+import (
+	"sort"
 
-// An index, typically an atom ID that defines the location of one side
+	"github.com/google/gapid/gapis/atom"
+)
+
+// An index defines the location of one side
 // synchronization dependency.
-type SynchronizationIndex uint64
+type SynchronizationIndex atom.ID
```

Comment:
Now that you've resolved the circular package dependency issue, do you still need the `[Synchronization]Index` type?